main.go

```diff
@@ -197,6 +197,10 @@ E.g. kairos-agent install-bundle container:quay.io/kairos/kairos...
 				EnvVars: []string{"REPOSITORY"},
 				Value:   "docker://quay.io/kairos/packages",
 			},
+			&cli.StringFlag{
+				Name:  "root_path",
```

Comment:
we used dashes `-` for `local-file` below. Maybe keep it consistent?

Reply:
good catch!